lamindb/_schema.py

```diff
@@ -295,4 +339,3 @@ def _get_related_name(self: Schema) -> str:
 
 Schema.members = members  # type: ignore
 Schema._get_related_name = _get_related_name
-Schema.feature_sets = Schema._artifacts_m2m  # backward compat
```

Comment:
This should have read `Schema.artifacts` instead of `Schema.feature_sets` to enable `schema.artifacts` pointing to the M2M.

But rather than this hacky solution, I want to properly redo the feature sets backward compatibility